packages/hint-ie-flexbox-compat/src/hint.ts

```diff
@@ -0,0 +1,65 @@
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { IHint } from 'hint/dist/src/lib/types';
+import { getFullCSSCodeSnippet, getCSSLocationFromNode } from '@hint/utils-css';
+import { StyleEvents, StyleParse } from '@hint/parser-css';
+import { Severity } from '@hint/utils-types';
+
+import meta from './meta';
+import { getMessage } from './i18n.import';
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class IEFlexboxCompatHint implements IHint {
+    public static readonly meta = meta;
+
+    public constructor(context: HintContext<StyleEvents>) {
+        if (!context.targetedBrowsers.some((browser) => {
+            return browser.toLowerCase().startsWith('ie ');
+        })) {
+            return;
+        }
+
+        let reported = false;
```

Comment:
In the VS Code extension or CLI in watch mode this will never get reset between runs (causing the error message to disappear). Add a 'scan::end' listener to the context to reset this.

Reply:
Thanks! Will do.